Don't hardcode path as it is architecture dependent

Debian Bug: #868285
Index: user-mode-linux/arch/um/drivers/port_user.c
===================================================================
--- user-mode-linux.orig/arch/um/drivers/port_user.c	2018-07-11 20:16:35.000000000 +0545
+++ user-mode-linux/arch/um/drivers/port_user.c	2018-07-20 12:09:49.725185686 +0545
@@ -168,7 +168,7 @@
 {
 	int new, err;
 	char *argv[] = { "/usr/sbin/in.telnetd", "-L",
-			 "/usr/lib/uml/port-helper", NULL };
+			 OS_LIB_PATH "/uml/port-helper", NULL };
 	struct port_pre_exec_data data;
 
 	new = accept(fd, NULL, 0);
